synapse/storage/schema/main/delta/74/01membership_tables_event_stream_ordering.sql

```diff
@@ -0,0 +1,21 @@
+/* Copyright 2022 Beeper
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ALTER TABLE current_state_events ADD COLUMN event_stream_ordering BIGINT;
+ALTER TABLE local_current_membership ADD COLUMN event_stream_ordering BIGINT;
+ALTER TABLE room_memberships ADD COLUMN event_stream_ordering BIGINT;
```

Comment:
it would be good to add a comment explaining what these columns are (ie, denormalised copies of the `stream_ordering` from the corresponding row in `events`). The `delta` files are pretty much the only place we have to document our schema.

Reply:
I'd love for us to use https://www.postgresql.org/docs/current/sql-comment.html or similar to put this into the schema! (Though it would be tricky to do that in a way that SQLite won't reject.)

Reply:
as a fun note, for `CREATE TABLE` etc statements, SQLite stores the SQL including comments, so comments within the statement get stored in the database automatically o.O

Reply:
In an ideal world, we'd have complete, up-to-date, commented, legible schema files... but we're a way off that.

Reply:
https://github.com/matrix-org/synapse/pull/15128/commits/bc3845c945f697f06e951577a89971f11bd505ec